.changeset/nasty-mayflies-sin.md

```diff
@@ -0,0 +1,5 @@
+---
+"modular-scripts": minor
+---
+
+Add "modular rename" command
```

Comment:
Let's add a tiny summary of what rename will do.

Reply:
Done, but still really tiny (a bigger description will be added in the next release notes)